src/Tribe/Modules/Maintenance.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Tribe\Extensions\EA_Additional_Options\Modules;

```

Comment:
See my comment on line 68 for this.
```suggestion
namespace Tribe\Extensions\EA_Additional_Options\Modules;

use TEC\Events\Custom_Tables\V1\Models\Occurrence;
```